service/matching/config/config.go

```diff
@@ -171,5 +174,6 @@ func NewConfig(dc *dynamicconfig.Collection, hostName string, getIsolationGroups
 		TaskDispatchRPSTTL:                   time.Minute,
 		MaxTimeBetweenTaskDeletes:            time.Second,
 		AllIsolationGroups:                   getIsolationGroups,
+		EnableStandbyTaskCompletion:          dc.GetBoolPropertyFilteredByTaskListInfo(dynamicconfig.MatchingEnableStandbyTaskCompletion),
```

Comment:
Minor suggestion / nit: 

It'll take a restart for this config to take effect, which might be a little slow if we need to stop it in a hurry. You could evaluate the dynamic-config per tasklist completion event and that way the responsiveness will be near-immediate:


```suggestion
		EnableStandbyTaskCompletion:          dc.GetBoolPropertyFilteredByTaskListInfo,
```